app/lib/features/home/widgets/space_chip.dart

```diff
@@ -65,21 +64,29 @@ class SpaceChip extends ConsumerWidget {
 
   Widget renderSpace(BuildContext context, SpaceItem space) {
     String spaceName = space.avatarInfo.displayName ?? space.roomId;
-    return InkWell(
-      onTap:
-          onTapOpenSpaceDetail ? () => goToSpace(context, space.roomId) : null,
-      child: useCompatView
-          ? Row(
-              children: [
-                const Text('In'),
-                ActerInlineTextButton(
-                  onPressed: () =>
-                      onTapSelectSpace != null ? onTapSelectSpace!() : null,
-                  child: Text(spaceName),
+    return useCompatView
+        ? Row(
+            children: [
+              Text(L10n.of(context).inKey),
+              const Text(' : '),
+              InkWell(
+                onTap: () => (onTapSelectSpace != null && !onTapOpenSpaceDetail)
+                    ? onTapSelectSpace!()
+                    : goToSpace(context, space.roomId),
+                child: Text(
+                  spaceName,
```

Comment:
Though technically speaking, the language could have this be grammatically incorrect. but as we have different design here this isn't easy to resolve. so let's ignore that.